environment.yml

```diff
@@ -15,6 +15,8 @@ dependencies:
   - geopandas
   - gdal
   - opencv
+  - s3fs
+  - pyarrow
```

Comment:
Where is `pyarrow` used?

Reply:
pyarrow is an optional dependency that you need to have in order for pandas to read parquet files.